CHANGELOG-1.1.md

```diff
@@ -0,0 +1,72 @@
+
+   * [v1.1.0](#v110)
+      * [Downloads for v1.1.0](#downloads-for-v110)
+         * [KubeEdge Binaries](#kubeedge-binaries)
+         * [Installer Binaries](#installer-binaries)
+         * [EdgeSite Binaries](#edgesite-binaries)
+      * [KubeEdge v1.1 Release Notes](#kubeedge-v11-release-notes)
+         * [1.1 What's New](#11-whats-new)
+         * [Known Issues](#known-issues)
+         * [Other notable changes](#other-notable-changes)
+
+# KubeEdge v1.1 Release Notes
+
+## 1.1 What's New
+
+**CSI support**
+
+This feature enables edged to support [Basic CSI Volume Lifecycle](https://github.com/container-storage-interface/spec/blob/master/spec.md#volume-lifecycle). And it's compatible with Kubernetes and CSI.
+
+**Admission webhook support**
+
+Admission webhook enables to mutate and add more validation for KubeEdge Api object, like device CRD. 
+
+**Upgrade kubernetes to v1.15.3**
+
+Upgrade the kubernetes to new version v1.15.3.
+
+**Add local up kubeedge script**
+
+A bash script that can stand up a KubeEdge cluster in a VM with just cloudcore, edgecore binaries and kind. It uses kind to start K8s cluster and runs cloudcore, edgecore binaries as process in VM.
+
+### Known Issues
+
+- Reliable message delivery is missing between cloud and edge.
+
+- Installer currently doesn't support installation of containerd, cni-plugins.
```

Comment:
Since we have an issue regarding keadm supporting installation of container-runtime is being discussed, we may not include this point as a known issue